pillar.example

```diff
@@ -1,10 +1,18 @@
 chrony:
   ntpservers:
-    - 'ntp0.nl.net'
-    - 'ntp1.nl.net'
-    - 'ntp2.nl.net'
-    - 'ntp.xs4all.nl'
-    - '0.nl.pool.ntp.org'
-    - '1.nl.pool.ntp.org'
-    - '2.nl.pool.ntp.org'
-    - '3.nl.pool.ntp.org'
+    - '0.debian.pool.ntp.org'
+    - '1.centos.pool.ntp.org'
+    - '2.arch.pool.ntp.org'
+    - '3.gentoo.pool.ntp.org'
+  keyfile: /etc/chrony.keys
+  allow: ['10/8', '192.168/16', '172.16/12' ]
+  otherparams: [ 'rtcsync',
+                 'makestep 10 3',
+                 'stratumweight 0',
+                 'bindcmdaddress 127.0.0.1',
+                 'bindcmdaddress ::1',
+                 'commandkey 1',
+                 'generatecommandkey',
+                 'noclientlog',
+                 'logchange 0.5',
+               ]
```

Comment:
This is only a suggestion but to help users customise TOFS, a commented block can be provided here, such as:

```yaml
  # tofs:
  #   The files_switch key serves as a selector for alternative
  #   directories under the formula files directory. See TOFS pattern
  #   doc for more info.
  #   Note: Any value not evaluated by `config.get` will be used literally.
  #   This can be used to set custom paths, as many levels deep as required.
  #   files_switch:
  #     - any/path/can/be/used/here
  #     - id
  #     - osfinger
  #     - os
  #     - os_family
  #   All aspects of path/file resolution are customisable using the options below.
  #   This is unnecessary in most cases; there are sensible defaults.
  #   path_prefix: template_alt
  #   dirs:
  #     files: files_alt
  #     default: default_alt
  #   source_files:
  #     chrony-config-file-file-managed:
  #       - 'chrony_alt.conf'
  #       - 'chrony_alt.conf.jinja'
```

Reply:
I have added the TOFS section within the example pillar file.